### PR TITLE
[dcp-551] Remove app from PyPi

### PR DESCRIPTION
**Pull request by ke4**
I found the following pypi related files in this repository that I am going to remove with this PR:

- Makefile
- README.md (version image from pypi)
- README.template (a section related to pypi)
- setup.cfg
- setup.py
- version
- Dockerfile

### CODE REVIEW

**Review by jacobwindsor — APPROVED** [review]
Amazing! Thanks a lot.

One thing, can you remove the existing package from PyPi? I guess we should. We're the only ones using it

**Review by jacobwindsor — CHANGES_REQUESTED** [review]
On second thoughts, are you sure this works?

The setup.py file installs a CLI tool AFAIK called ingest-graph-validator. Without that, this won't be functional

**Comment by ke4** [comment]
> On second thoughts, are you sure this works?
> 
> The setup.py file installs a CLI tool AFAIK called ingest-graph-validator. Without that, this won't be functional

Thanks for this comment. I fixed this issue and I hope in the right way. Please check it again. I tested it from a clean environment and it worked for me. Tx

File: Dockerfile [hunk 1]
```diff
@@ -17,6 +17,7 @@ RUN pip install -r requirements.txt
 
 ADD . /ingest-graph-validator/
 RUN pip install -e .
+RUN pip install -r requirements.txt
```
Comment (jacobwindsor) [inline Dockerfile:20]: Shouldn't it still be installed with setuptools?

**Review by jacobwindsor — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (ke4) [reply Dockerfile]: You are totally right. I changed `setup.py` and moved every dependency setup there. I tested it and it worked for me.

**Review by jacobwindsor — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -48,26 +46,37 @@ pip install -e .
 
 1. Ensure Docker is installed and running
 
-1. `docker run -p7687:7687 -p7474:7474 --env NEO4J_AUTH=neo4j/password --env=NEO4J_ACCEPT_LICENSE_AGREEMENT=yes neo4j:3.5.14-enterprise` (in another terminal session or with `-d` (detached) flag)
+2. Create a file for the GCP credentials in your HOME folder: `~/.secrets/gcp_credentials`
```
Comment (amnonkhen) [inline README.md:49]: Have you considered creating a setup script with the commands to create the directory and download the secret?

e.g.

```bash
mkdir ~/.secrets
aws secretsmanager get-secret-value --secret-id ingest/${ENV}/gcp-credentials.json  --region=us-east-1
```

File: setup.py [hunk 3]
```diff
@@ -16,46 +10,22 @@
...
-        'console_scripts': [
-            "ingest-graph-validator=ingest_graph_validator.ingest_graph_validator:entry_point",
-        ],
+        "console_scripts": [
+            "ingest-graph-validator = ingest_graph_validator.ingest_graph_validator:entry_point"
+        ]
     },
     install_requires=[
-        'hca-ingest @ git+https://github.com/ebi-ait/ingest-client.git@782dd754#egg=hca_ingest',
+        'hca-ingest @ git+https://github.com/ebi-ait/ingest-client.git@7fd3c25#egg=hca_ingest',
```
Comment (aaclan-ebi) [inline setup.py:25]: might be better to just publish a new version

**Review by aaclan-ebi — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (ke4) [reply README.md:49]: @amnonkhen The task was to remove this app from pypi. The above you are asking is not belongs to that task.

**Pull request merged**